include/curl/curl.h

```diff
@@ -1875,6 +1875,9 @@ typedef enum {
   /* Specify URL using CURL URL API. */
   CINIT(CURLU, OBJECTPOINT, 282),
 
+  /* Set the IP-Address string to use as outgoing IP Addr */
+  CINIT(LOCALADDR, OBJECTPOINT, 283),
```

Comment:
s/OBJECTPOINT/STRINGPOINT/

We use STRINGPOINT for options that accept a C string passed to them.